### PR TITLE
Add list of OME 2021 speakers

### PR DESCRIPTION
**Pull request by sbesson**
- add all speakers who accepted the invite as `perfomer` of the appropriate `Event`
- expand Overview page
- remove Lightning talks page - might be integrated as part of the day

### CODE REVIEW

File: _data/ome2021.json [hunk 1]
```diff
@@ -273,10 +355,56 @@
...
+          "name": "Luca Lianas",
+          "affiliation": {
+            "@type": "Organization",
+            "name": "CRS4"
+          },
+          "sameAs": "https://www.crs4.it/peopledetails/219/luca-lianas"
+        },
+        {
+          "@type": "Person",
+          "name": "Paul Pfelts",
```
Comment (jburel) [inline _data/ome2021.json:388]: Felts (extra P)

Comment (sbesson) [reply _data/ome2021.json:388]: :+1: (feel free to use suggestions)

```suggestion
          "name": "Paul Felts",
```

**Comment by pwalczysko** [comment]
Screenshot 2021-04-23 at 11 32 25

The double-usage of the term "overview" here could be improved, afaik, these two overviews are different, describing different entities (overview of the page/event vs a distinct programme item on the conference schedule.

For the first "Overview" (the tab name) I would suggest a "General Information" header ?

**Comment by sbesson** [comment]
Good point @pwalczysko the double usage is certainly confusing. I guess one of the issue is that we don't know what this first `Overview` slot on day 1 is really about yet. Since we understand the Keynote will happen on the afternoon, an alternative to your proposal would be to call it `Introduction` i.e. `Introduction, Keynote and Demos`

/cc @jburel @joshmoore

**Comment by pwalczysko** [comment]
> Good point @pwalczysko the double usage is certainly confusing. I guess one of the issue is that we don't know what this first `Overview` slot on day 1 is really about yet. Since we understand the Keynote will happen on the afternoon, an alternative to your proposal would be to call it `Introduction` i.e. `Introduction, Keynote and Demos`
> 
> /cc @jburel @joshmoore

Definitely like ``Introduction``. Whatever that is in the end, it really makes an impression to be on the right place at the right time in the programme.

File: _data/ome2021.json [hunk 2]
```diff
@@ -13,79 +13,94 @@
...
-      }
+          "location": {
+            "@type": "VirtualLocation",
+            "url": "TBD"
+          }
         },
         {
           "@type": "Event",
-          "name": "TBD",
+          "name": "Keynote",
```
Comment (pwalczysko) [inline _data/ome2021.json:94]: Keynote speaker ?
Or maybe
Keynote lecture ? 
In combination with a concrete name below, the fuller title would sound better imho.

**Comment by pwalczysko** [comment]
@sbesson opened https://github.com/sbesson/www.openmicroscopy.org/pull/17 to fix the overlap 

Before
Screenshot 2021-04-23 at 12 23 33

After
Screenshot 2021-04-23 at 12 24 05

File: _data/ome2021.json (hunk 2 above)
Comment (sbesson) [reply _data/ome2021.json:94]: I think the `Keynote speaker` is the `perfomer` of the `Event`. For the name, `Keynote Lecture` makes sense. Should I add `TBD` or `Title TBD` as the description for now?

Comment (pwalczysko) [reply _data/ome2021.json:94]: > Should I add TBD or Title TBD as the description for now?

Yes, Title TBD would, I think, make it even more solid.

**Comment by pwalczysko** [comment]
LGTM

**Review by pwalczysko — APPROVED** [review]

**Comment by sbesson** [comment]
Thanks. Merging this and deploying it on the website to get some preliminary information about the speakers. Next steps:

- add more blurb similar to the ones in #472
- add new speakers
- link to speaker pictures

**Pull request merged**